ycsb-mongodb/mongodb/src/main/java/com/yahoo/ycsb/db/MongoDbClient.java

```diff
@@ -128,7 +128,8 @@ enum Encryption {
     private static Encryption encryptionType = Encryption.UNENCRYPTED;
     private static ArrayList<Long> contentionFactors;
     private static HashMap<String, DiscreteGenerator> discreteFields;
-
+    private static String cryptSharedLibPath = "";
```

Comment:
I'm not crazy about all private static variables given that you could pass the properties object to generateEncryptionSettings and have it validate the config and access the propery values directly.

Reply:
fixed. new patch: https://spruce.mongodb.com/version/631f58f02fbabe310e95d23d/tasks